File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
import sys
input = sys.stdin.readline

data = list(map(int, input().split()))

cnt = set(data)

# 같은 눈 3개
if len(cnt) == 1:
    print(10000 + data[0] * 1000)
# 같은 눈 2개
elif len(cnt) == 2:
    # 같은 눈 찾기
    for i in cnt:
        if data.count(i) == 2:
            print(1000 + i * 100)
            break
# 같은 눈 없음
else:
    print(max(data) * 100)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');

let [a, b, c] = input[0].split(' ').map(Number);

if (a === b) {
  if (a === c) {
    console.log(10000 + a * 1000);
  } else {
    console.log(1000 + a * 100);
  }
} else if (b === c) {
  if (b === a) {
    console.log(10000 + b * 1000);
  } else {
    console.log(1000 + b * 100);
  }
} else if (a === c) {
  if (a === b) {
    console.log(10000 + a * 1000);
  } else {
    console.log(1000 + a * 100);
  }
} else {
  console.log(Math.max(a, b, c) * 100);
}
